setup_demo.sh

```
@@ -0,0 +1,28 @@
+# Quick setup script for demo - resets everything and assigns a fresh challenge
+
+echo "🎬 Setting up fresh demo environment..."
+echo ""
+
+# Reset demo data
+echo "1️⃣  Resetting demo data..."
+poetry run python reset_demo_data.py --username "${1:-test_challenge_user}" 2>&1 | grep -E "(✅|❌|📊|Using user|Resetting)"
+
+echo ""
+echo "2️⃣  Assigning new challenge..."
+poetry run python manage.py generate_daily_challenges --date $(date +%Y-%m-%d) 2>&1 | tail -3
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)